boot/src/test/java/com.taobao.arthas.boot/DownloadUtilsTest.java

```diff
@@ -0,0 +1,42 @@
+package com.taobao.arthas.boot;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.ArrayList;
+
+public class DownloadUtilsTest {
+
+    @Test
+    public void testreadMavenReleaseVersion() {
```

Comment:
As discussed, capitalize the first letter in the method name to separate it from the "test" prefix.

Reply:
first letters of the method names have been capitalised